spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -134,6 +134,18 @@
 	 */
 	public static final String SPAN_PEER_SERVICE_TAG_NAME = "peer.service";
 
+	/**
+	 * ID of the instance from which the span was originated. This tag is related to the
+	 * client side communication of an RPC call.
+	 */
+	public static final String SPAN_CLIENT_INSTANCEID = "client.instanceid";
```

Comment:
I don't think you need to add this twice, if you think about how it is used.

For example, are you going to search for a client whose instanceid is this? or anything whose instanceid is this? In the normal dual-host span model, the span detail screen would show the servicename and IP address who reported the instanceid (which would also match any other client-related tags). In any future "single-host span" model, this clarifier will become even more redundant. In other words, unless there's a very good reason to qualify this as client/server I'd just leave it as a normal tag like "spring.application.instanceid" or shorter like "spring.instanceid" id there's not likely an instanceid that isn't an application